base/applications/mstsc/lang/ro-RO.rc

```diff
@@ -65,6 +67,6 @@ BEGIN
     IDS_PIXEL "%lux%lu pixeli"
     IDS_FULLSCREEN "Pe tot ecranul"
     IDS_BROWSESERVER "<Căutare mai multe…>"
-    IDS_HEADERTEXT1 "Remote Desktop"
-    IDS_HEADERTEXT2 "Connection"
+    IDS_HEADERTEXT1 "Birou de la distanță"
```

Comment:
To be frank I'm not keen on literal translations of words between languages. Whilst `desk` means `birou`, that's for sure, `desktop` in question actually refers to `PC`, `calculator` or just simply desktop as the average Romanian speaker with basic of understanding of tech knows what does this stuff mean. `Birou` gives weird vibes to be honest.

Either translate it as `Desktop de la distanță` or just `Calculator de la distanță`. I would opt more for the former suggestion. I do not think we are supposed to translate everything to Romanian just for the sake of it.

Reply:
I also agree for this particular suggestion - „calculator la distanță” sounds better adapted (besides translated). Now, there is a bit of work to touch the rest of the Romanian resources where it still is translated like that.
Just for the record, the simple „Desktop” I still find better translated as „Birou”.